### PR TITLE
Patch for encrypting db if found unencrypted

### PR DESCRIPTION
**Pull request by wmathurin**

### CODE REVIEW

File: GlobalStoreTester530fixed/0001-Encrypt-db-if-found-unencrypted-5.3-bug-fix.patch [hunk 1]
```diff
@@ -0,0 +1,42 @@
+From 8a1f4bf1b78e6e6853332073aaccb12d89e43f0d Mon Sep 17 00:00:00 2001
```
Comment (wmathurin) [inline GlobalStoreTester530fixed/0001-Encrypt-db-if-found-unencrypted-5.3-bug-fix.patch:1]: That's the patch for the customer.

File: GlobalStoreTester530fixed/install.js [hunk 2]
```diff
@@ -12,9 +12,13 @@ for (var sdkDependency in packageJson.sdkDependencies) {
     var targetDir = path.join('mobile_sdk', sdkDependency);
     if (fs.existsSync(targetDir))
         console.log(targetDir + ' already exists - if you want to refresh it, please remove it and re-run install.js');
-    else
+    else {
         execSync('git clone --branch ' + branch + ' --single-branch --depth 1 ' + repoUrl + ' ' + targetDir, {stdio:[0,1,2]});
+        execSync('git apply ../../0001-Encrypt-db-if-found-unencrypted-5.3-bug-fix.patch', {stdio:[0,1,2], cwd:'mobile_sdk/salesforcemobilesdk-ios'});
```
Comment (wmathurin) [inline GlobalStoreTester530fixed/install.js:17]: Here the patch is getting applied.

**Comment by wmathurin** [comment]
@bhariharan

**Review by bhariharan — APPROVED** [review]
LGTM! 👍

**Pull request merged**